examples/patterns/pagination.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Pagination
+category: _patterns
+---
+
+<footer class="p-pagination">
+  <a class="p-pagination__link" href="#previous">
+    <span class="p-pagination__arrow">
+      <img src="https://assets.ubuntu.com/v1/e8d2e45f-chevron-left.svg" alt="">
```

Comment:
We need to make all patterns work offline. To be consistent with other patterns with permanent image styling can you add these to a pseudo selector of the links as base64 backgrounds, please.